tests/ui/const-generics/transmute_no_gate.stderr

```diff
@@ -4,80 +4,80 @@ error[E0512]: cannot transmute between types of different sizes, or dependently-
 LL |     std::mem::transmute(v)
    |     ^^^^^^^^^^^^^^^^^^^
    |
-   = note: source type: `[[u32; H]; W]` (this type does not have a fixed size)
-   = note: target type: `[[u32; W]; H]` (this type does not have a fixed size)
+   = note: source type: `[[u32; H]; W]` (`[[u32; H]; W]` is too generic to determine the layout)
```

Comment:
I prefer the old wording of "does not have a fixed size" here (but this could say ``(`[[u32; H]; W]` does not have a fixed size)``).

"too generic" is a term that makes sense to us compiler developers, but for someone new to the language or maybe even programming in general it doesn't mean anything.

Reply:
Since the compiler has this information I would like to report it in some way.
What if `TooGeneric` generated: `T has no fixed size; too generic`?

Reply:
This formatting with a ; doesn't really fit with the other errors the compiler reports.

If you want to incorporate that X is a generic in the error message, maybe something like `[[u32; H]; W] does not have a fixed size, because H and W are generic parameters` could work, but that seems a bit wordy to me and it will only get worse if there are many generic parameters.

IMO `` `[[u32; H]; W]` does not have a fixed size `` is perfectly clear and mentioning generics here doesn't add any real value.

Reply:
All right, let's go with ``T` does not have a fixed size`